primitives/src/lib.rs

```diff
@@ -283,6 +283,8 @@ sp_api::decl_runtime_apis! {
             session: SessionIndex
         ) -> Result<SessionCommittee<AccountId>, SessionValidatorError>;
         fn next_session_aura_authorities() -> Vec<AuraAuthorityId>;
+        /// Allows to get owner's AccountId for an Aleph key used in the current session.
+        fn aleph_key_owner(key: AuthorityId) -> Option<AccountId>;
```

Comment:
Is the `aleph_` prefix necessary? I understand that it's b/c this is from the "aleph sesion" but I haven't heard such concept before (maybe my lack of knowledge) and it seems that no other method in this API (`AlephSessionAPI`) is prefixed with it either.

Reply:
Yup, I wanted to distinguish `aleph_key` from `aura_key`, but this is rather unnecessary in `aleph` api.